packages/react-components/src/molecules/__tests__/MembersList.test.tsx

```diff
@@ -38,21 +38,16 @@ it('only show lab information if the user is on a lab', async () => {
       members={[
         {
           ...createListUserResponse(1).items[0],
-          displayName: 'Bat Man',
-          role: 'Boss',
-          teams: [],
-          labs: [
-            { id: 'cd7be4904', name: 'Manchester' },
-            { id: 'cd7be4905', name: 'Glasgow' },
-          ],
+          firstLine: 'Bat Man',
```

Comment:
We should update the description of this test to reflect the new logic 
`thirdLine` shows when there available data